scripts/fork-and-cast-spell.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -e
+
+trap "kill 0" EXIT
```

Comment:
You should add `unset ETH_FROM` as well here. I have this environment variable set and it is messing with the submission.

Reply:
👍 --from passed to avoid issues with different ETH_FROM & private key